base/reflection.jl

```diff
@@ -542,14 +546,18 @@ code_llvm(f::ANY, types::ANY=Tuple) = code_llvm(STDOUT, f, types)
 code_llvm_raw(f::ANY, types::ANY=Tuple) = code_llvm(STDOUT, f, types, false)
 
 """
-    code_native([io], f, types)
+    code_native([io], f, types, [asm_variant])
 
 Prints the native assembly instructions generated for running the method matching the given
-generic function and type signature to `io` which defaults to `STDOUT`.
+generic function and type signature to `io` which defaults to `STDOUT`.Switch assembly syntax using `asmvariant` symbol parameter set to `:att` for AT&T syntax or `:intel` for Intel syntax. Output is AT&T syntax by default .
```

Comment:
Line wrap this. Space (or newline) right before `Switch`, no space right before final period